Assets/SEE/Game/SceneManipulation/GameNodeMarker.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using SEE.GO;
+using SEE.Utils;
+using UnityEngine;
+
+namespace SEE.Game.SceneManipulation
+{
+    public static class GameNodeMarker
+    {
+        public static GameObject AddMarker(GameObject parent) {
+
+            // When there is no marker yet, one will be created
+            if(parent.transform.Find("Sphere")==null)
+            {
+                Debug.Log("There's no marker yet!");
+                GameObject result = GameObject.CreatePrimitive(PrimitiveType.Sphere);
+                result.GetComponent<Renderer>().material.color = Color.yellow;
+
+                // minimum of the width (x axis) and depth (z axis) of the marked node for diameter of sphere
+                float diameterSphere = Math.Min(parent.transform.lossyScale.x, parent.transform.lossyScale.z);
+                result.transform.localScale = new Vector3(diameterSphere, diameterSphere, diameterSphere);
```

Comment:
You could also write `result.transform.localScale = Vector3.one * diameterSphere`.